.travis.yml

```diff
@@ -136,7 +136,7 @@ jobs:
 
 stages:
   - test
-  - name: sonar
-    if: branch = master AND type != pull_request
-  - name: deploy
-    if: tag IS present
+  # - name: sonar
+  #   if: branch = master AND type != pull_request
+  # - name: deploy
+  #   if: tag IS present
```

Comment:
@BridgeAR Mentioning a stage inside the stages only ensures the stages will run in order from top to bottom and each stage only runs if the conditions evaluate to true. So the `sonar` and `deploy` stages still run when commented out.

What you actually want is something like this, which only deactivates the sonar branch.
```
stages:
  - test
  - name: sonar
    if: false AND branch = master AND type != pull_request
  - name: deploy
    if: tag IS present
```